app/controllers/administrateurs/procedures_controller.rb

```diff
@@ -264,9 +264,14 @@ def publication
           draft_revision: :types_de_champ
         ).find(params[:procedure_id])
 
-      @procedure.path = @procedure.suggested_path(current_administrateur)
-      @current_administrateur = current_administrateur
-      @closed_procedures = current_administrateur.procedures.with_discarded.closes.map { |p| ["#{p.libelle} (#{p.id})", p.id] }.to_h
+      if @procedure.auto_archive_on && !@procedure.auto_archive_on.future?
```

Comment:
```suggestion
      if @procedure.auto_archive_on && @procedure.auto_archive_on.past?
```

ruby, rails, c'est magique

Reply:
J'y avais pensé :)  Mais je voulais que ca soit en erreur pas uniquement pour le passé, également si auto_archive_on vaut la date du jour en cours.
Dit autrement, `!future?` == past? || Date.current`